internal/processor/awscontainerinsightprocessor/stores/sampleMountFile/mounts

```diff
@@ -0,0 +1,2 @@
+/dev/nvme1n1 /var/lib/kubelet/plugins/kubernetes.io/aws-ebs/mounts/aws/us-west-2b/vol-0d9f0816149eb2050 ext4 rw,relatime,data=ordered 0 0
+/dev/nvme1n1 /var/lib/kubelet/pods/df570351-2e4c-11e9-95ea-0a695d7ce286/volumes/kubernetes.io~aws-ebs/pvc-df563cf6-2e4c-11e9-95ea-0a695d7ce286 ext4 rw,relatime,data=ordered 0 0
```

Comment:
is this file still needed? is it just for testing?

Reply:
It's for testing only. I will remove it for now. When we send PR to upstream repo `opentelemetry-collector-contrib`, we will need to add more unit tests and might need this file.